data/data/install.openshift.io_installconfigs.yaml

```
@@ -5103,9 +5126,27 @@ spec:
                           type: object
                         zone:
                           description: |-
-                            zone defines a VSpherePlatformFailureDomain which
-                            includes the name of the vCenter tag, the failure domain type
-                            and the name of the vCenter tag category.
+                            The zone is the name of the tag in vCenter that is associated with
+                            the tag category `openshift-zone`. The zone name must match the tag name
+                            and must exist prior to installation. When zoneType is HostGroup the
+                            ESXi hosts defined in the provided in the topology.hostGroup field must be tagged.
+                            When the zoneType is ComputeCluster the tag must be attached to the topology.computeCluster
+                            object in vCenter.
+                          type: string
+                        zoneType:
+                          allOf:
+                          - enum:
+                            - HostGroup
+                            - Datacenter
+                            - ComputeCluster
+                          - enum:
+                            - ComputeCluster
+                            - HostGroup
+                          description: |-
+                            When zoneType is ComputeCluster the regionType must be Datacenter
+                            When zoneType is HostGroup the regionType must be HostGroup
```

@jcpowermac should be ->
```suggestion
                            When zoneType is HostGroup the regionType must be ComputeCluster
```
right?